samples/Databricks-AzureSQL/README.md

```diff
@@ -1,23 +1,28 @@
+
+The objective of this solution is to demonstrate the integration between Azure Databricks and Azure SQL Managed Instance to deliver insights and data visualizations using a publicly available [COVID-19 dataset](https://azure.microsoft.com/en-au/services/open-datasets/catalog/ecdc-covid-19-cases/). While Azure Databricks provides the distributed computing power to process and transform complex datasets, Azure SQL Managed Instance acts as a recipient of the batch scored predictions and transformed data to serve it to business users. In doing so, the solution uses a publicly available COVID-19 dataset and runs a machine learning model in Databricks to predict the fatalities which are then written into a datamart in Azure SQL Managed Instance for visualization and reporting.
 
 > Note: Gradient Boosting Trees (GBT) Regression model has been used in
 > the solution purely for demonstration purposes and by no means should
 > this be considered as the only machine learning algorithm to run
 > predictions on the dataset used.
 
 
-**Azure SQL Spark Connector**
-
-The [Spark connector](https://docs.microsoft.com/en-us/azure/azure-sql/database/spark-connector) enables databases in Azure SQL Database, Azure SQL Managed Instance, and SQL Server to act as the input data source or output data sink for Spark jobs. The Spark connector supports Azure Active Directory (Azure AD) authentication to connect to Azure SQL Database and Azure SQL Managed Instance and it utilizes the Microsoft JDBC Driver for SQL Server to move data between Spark worker nodes and databases.
+**Apache Spark Connector for SQL Server and Azure SQL**
 
-The connector can either be downloaded from the [azure-sqldb-spark GitHub repo](https://github.com/Azure/azure-sqldb-spark/tree/master/releases/azure-sqldb-spark-1.0.0) or imported into Azure Databricks using the Maven coordinate: `com.microsoft.azure:azure-sqldb-spark:1.0.2`
+The [Spark connector for SQL Server and Azure SQL](https://docs.microsoft.com/en-us/azure/azure-sql/database/spark-connector) enables databases in Azure SQL Database, Azure SQL Managed Instance, and SQL Server to act as the input data source or output data sink for Spark jobs. The Spark connector is a high-performance connector that enables you to use  transactional data in big data analytics and persists results for ad-hoc queries or reporting.
```

Comment:
Sorry one last change -  this docs link is to the Azure SQL Spark connector. Note that this one is different from this project. We do not have a docs page for this connector as of now other than a page in the SQL Server Big Data Clusters docs

Reply:
Done and committed

Reply:
Thanks!